articles/install-guide/setup/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Setup Overview
+author: gillenhaalb
+ms.author: gillenhaalb@gmail.com
+ms.date: 7/19/2019
+ms.topic: article
+uid: microsoft.quantum.install.setup
+---
+
+# Setup Overview #
+
+To begin writing and running quantum programs on your own computer, follow these instructions for installing the Quantum Development Kit.
+Whether on Windows, Linux, or macOS, you will need the right machinery in the background, so the first step below is to download the [.NET Core SDK](https://dotnet.microsoft.com/).
+Then, our instructions will guide you through installing the Quantum Devleopment Kit for whichever classical host environment and language you prefer--the choice is yours, as the power of Q# will be at your fingertips regardless!
+
+## Install .NET Core SDK ##
+
+The Quantum Development Kit uses the [.NET Core SDK](https://dotnet.microsoft.com/) to make it easy to create, build, and run Q# projects from the command line.
+Whatever your preferred editor or environment may be, to start working with Q# on your own computer, you will need the latest version (2.2 or later) installed.
+You can do this by following the instructions from the [.NET downloads page](https://www.microsoft.com/net/download).
+
+## Install the Quantum Development Kit for Your Preferred Environment ##
+
+### Visual Studio Editors and the Command Line ###
+
+With the .NET Core SDK now installed, you can implement Q# programs in editors such as Visual Studio Code and Visual Studio, or even directly in the command line.
```

Comment:
I would remove this line, since .NET core does not allow Q# development, they need to install the QDK first

Reply:
For users writing host programs in C#, installing the .NET Core SDK is enough in some sense, though I would definitely recommend installing any applicable IDE extensions and the project templates. The latter should be made a bit less necessary by the New Project... command that was added in Microsoft/qsharp-compiler#79, but it's still helpful.